kong/README.md

```diff
@@ -30,7 +30,7 @@ Restart the Agent to begin sending Kong metrics to Datadog.
 
 ### Validation
 
-Run the Agent's `info` subcommand and look for kong` under the Checks section:
+Run the Agent's `info` subcommand and look for kong under the Checks section:
```

Comment:
The other README highlight the check's name: I think we should add the missing quote before `kong` instead of removing it.

Reply:
good catch